docs/dashboards/reports-and-components.md

```diff
@@ -0,0 +1,22 @@
+title: Sematext Dashboards Reports and Components
+description: Sematext Cloud is a modern monitoring, log management, transaction tracing, and real user monitoring system that includes over 40 monitoring integrations. It is a suite of products that combine high-quality logging experience with other monitoring and alerting devops tools helping fix IT production issues
```

Comment:
Maybe not use "over 40" because that is outdated and any such specific number will always get outdated sooner or later.